src/Controller/ConfigController.php

```diff
@@ -319,14 +323,85 @@ public function getAction(Request $request, Service $graphQlService): JsonRespon
             $config['schema']['specialEntities'] = [];
         }
 
-        $specialSettings = ["document", "document_folder", "asset", "asset_folder", "asset_listing", "object_folder"];
-        foreach ($specialSettings as $key) {
-            if (!isset($config['schema']['specialEntities'][$key])) {
-                $config['schema']['specialEntities'][$key] = ["id" => $key];
-            }
+        $coreSettings = [
+            new SpecialEntitySetting(
+                'document',
+                true,
+                false,
+                false,
+                true,
+                $config['schema']['specialEntities']['document']['read'] ?? false,
+                $config['schema']['specialEntities']['document']['create'] ?? false,
+                $config['schema']['specialEntities']['document']['update'] ?? false,
+                $config['schema']['specialEntities']['document']['delete'] ?? false
+            ),
+            new SpecialEntitySetting(
+                'document_folder',
+                true,
+                false,
+                false,
+                true,
+                $config['schema']['specialEntities']['document_folder']['read'] ?? false,
+                $config['schema']['specialEntities']['document_folder']['create'] ?? false,
+                $config['schema']['specialEntities']['document_folder']['update'] ?? false,
+                $config['schema']['specialEntities']['document_folder']['delete'] ?? false
+            ),
+            new SpecialEntitySetting(
+                'asset',
+                true,
+                true,
+                true,
+                true,
+                $config['schema']['specialEntities']['asset']['read'] ?? false,
+                $config['schema']['specialEntities']['asset']['create'] ?? false,
+                $config['schema']['specialEntities']['asset']['update'] ?? false,
+                $config['schema']['specialEntities']['asset']['delete'] ?? false
+            ),
+            new SpecialEntitySetting(
+                'asset_folder',
+                true,
+                true,
+                true,
+                true,
+                $config['schema']['specialEntities']['asset_folder']['read'] ?? false,
+                $config['schema']['specialEntities']['asset_folder']['create'] ?? false,
+                $config['schema']['specialEntities']['asset_folder']['update'] ?? false,
+                $config['schema']['specialEntities']['asset_folder']['delete'] ?? false
+            ),
+            new SpecialEntitySetting(
+                'asset_listing',
+                true,
```

Comment:
create update delete is possible for listings ?

Reply:
well, actually not, but that wasn't limited before this PR also.